packages/twenty-front/src/modules/workspace/components/WorkspaceMemberCard.tsx

```diff
@@ -28,30 +31,50 @@ const StyledEmailText = styled.span`
 `;
 
 type WorkspaceMemberCardProps = {
-  workspaceMember: WorkspaceMember;
+  workspaceMember: WorkspaceMember | WorkspaceInvitation;
   accessory?: React.ReactNode;
 };
 
+const isWorkspaceInvitationGuard = (
+  workspaceMember: WorkspaceMemberCardProps['workspaceMember'],
+): workspaceMember is WorkspaceInvitation => 'expiresAt' in workspaceMember;
+
 export const WorkspaceMemberCard = ({
   workspaceMember,
   accessory,
-}: WorkspaceMemberCardProps) => (
-  <StyledContainer>
-    <Avatar
-      avatarUrl={workspaceMember.avatarUrl}
-      placeholderColorSeed={workspaceMember.id}
-      placeholder={workspaceMember.name.firstName || ''}
-      type="squared"
-      size="xl"
-    />
-    <StyledContent>
-      <OverflowingTextWithTooltip
-        text={
-          workspaceMember.name.firstName + ' ' + workspaceMember.name.lastName
+}: WorkspaceMemberCardProps) => {
+  const isWorkspaceInvitation = isWorkspaceInvitationGuard(workspaceMember);
+
+  return (
+    <StyledContainer>
+      <Avatar
+        avatarUrl={isWorkspaceInvitation ? null : workspaceMember.avatarUrl}
+        placeholderColorSeed={workspaceMember.id}
+        placeholder={
+          isWorkspaceInvitation
+            ? workspaceMember.email[0]
```

Comment:
does workpaceMember have multiple emails?

Reply:
workspaceMember.primaryEmail?

Reply:
This file doesn't exist anymore in the milestone 2